src/general/SpeciesInfo.cs

```diff
@@ -0,0 +1,11 @@
+/// <summary>
+///   Class that stores species-wide information.
+///   It can be expanded to have any species parameter we might want to keep,
+///   e.g. base structure, behavioral values... and such to draw species history.
+/// </summary>
+/// <remarks> This class does not account for individual variations!</remarks>
```

Comment:
What does `does not account for individual variations!` mean? I have no clue. This needs more explanation...